test/linalg/rowvector.jl

```diff
@@ -1,318 +0,0 @@
-
-    @test (rv*Adjoint(ut))::RowVector == [2 6 12]
-    @test (ut*rvadjoint(rv))::Vector == [2,6,12]
-
-    @test_throws DimensionMismatch rvadjoint(rv)*Adjoint(ut)
-    @test (Adjoint(ut)*rvadjoint(rv))::Vector == [2,6,12]
-
-    @test_throws DimensionMismatch Adjoint(ut)*rv
-    @test_throws DimensionMismatch rvadjoint(rv)*ut
-
-    @test (rv/ut)::RowVector ≈ [2/1  3/2  4/3]
-    @test (rv/Transpose(ut))::RowVector ≈ [2/1  3/2  4/3]
-    @test (rv/Adjoint(ut))::RowVector ≈ [2/1  3/2  4/3]
-
-    @test_throws DimensionMismatch ut\rv
-end
-
-@testset "Symmetric/Hermitian ambiguity methods" begin
-    S = Symmetric(rand(3, 3))
-    H = Hermitian(rand(3, 3))
-    v = (rvadjoint(rand(3)))::RowVector
-    @test_throws DimensionMismatch S\v
-    @test_throws DimensionMismatch H\v
-end
-
-# issue #20389
-@testset "1 row/col vec*mat" begin
-    let x=[1,2,3], A=ones(1,4), y=rvadjoint(x), B=adjoint(A), C=x.*A
-        @test x*A == rvadjoint(y)*A == x*Adjoint(B) == rvadjoint(y)*Adjoint(B) == C
-        @test Adjoint(A)*rvadjoint(x) == Adjoint(A)*y == B*rvadjoint(x) == B*y == adjoint(C)
-    end
-end
-@testset "complex 1 row/col vec*mat" begin
-    let x=[1,2,3]*im, A=ones(1,4)*im, y=rvadjoint(x), B=adjoint(A), C=x.*A
-        @test x*A == rvadjoint(y)*A == x*Adjoint(B) == rvadjoint(y)*Adjoint(B) == C
-        @test Adjoint(A)*rvadjoint(x) == Adjoint(A)*y == B*rvadjoint(x) == B*y == adjoint(C)
-    end
-end
-
-@testset "issue #20979" begin
-    f20979(z::Complex) = [z.re -z.im; z.im z.re]
-    v = rvadjoint([1+2im])
-    @test (f20979.(v))[1] == f20979(v[1])
-    @test f20979.(v) == f20979.(collect(v))
-
-    w = rand(ComplexF64, 3)
-    @test f20979.(rvadjoint(v)) == f20979.(collect(rvadjoint(v))) == rvadjoint(f20979.(v))
-
-    g20979(x, y) = [x[2,1] x[1,2]; y[1,2] y[2,1]]
-    v = [rand(2,2), rand(2,2), rand(2,2)]
-    @test g20979.(rvadjoint(v), rvadjoint(v)) == g20979.(collect(rvadjoint(v)), collect(rvadjoint(v))) ==
-          map(g20979, rvadjoint(v), rvadjoint(v)) == map(g20979, collect(rvadjoint(v)), collect(rvadjoint(v)))
-end
-
-@testset "ambiguity between * methods with RowVectors and ConjRowVectors (#20971)" begin
-    @test RowVector(ConjArray(ones(4))) * ones(4) == 4
-end
-
-@testset "setindex!/getindex" begin
-    v = [2, 3, 4]
-    rv = rvtranspose(v)
-    @test_throws BoundsError setindex!(rv, 5, CartesianIndex((5, 4, 3)))
-    rv[CartesianIndex((1, 1, 1))] = 5
-    @test_throws BoundsError getindex(rv, CartesianIndex((5, 4, 3)))
-    @test rv[1] == 5
-
-    @test rv[:, 2]::Vector == [v[2]]
-    @test rv[:, 2:3]::RowVector == rvtranspose(v[2:3])
-    @test rv[:, :]::RowVector == rv
-
-    v = [1]
-    rv = rvtranspose(v)
-    rv[CartesianIndex()] = 2
-    @test rv[CartesianIndex()] == 2
-    rv[CartesianIndex(1)] = 1
-    @test rv[CartesianIndex(1)] == 1
-
-    # setindex!(v::RowVector, ...) should return v rather than v's parent
-    @test setindex!(RowVector([1, 2, 3]), 4, 1)::RowVector == [4 2 3]
-end
```

Comment:
Equivalents of almost all of these tests (and more besides) now exercise `Adjoint`/`Transpose`.